src/libs/localStore/userPersistedStorage.spec.ts

```diff
@@ -0,0 +1,61 @@
+import CozyClient from 'cozy-client'
+
+import AsyncStorage from '@react-native-async-storage/async-storage'
+
+import {
+  getUserPersistedData as initialGetUserPersistedData,
+  storeUserPersistedData as initialStoreUserPersistedData
+} from '/libs/localStore/userPersistedStorage'
+
+const getUserPersistedData = initialGetUserPersistedData as (
+  client: CozyClient,
+  name: string
+) => Promise<unknown>
+const storeUserPersistedData = initialStoreUserPersistedData as (
+  client: CozyClient,
+  name: string,
+  value: unknown
+) => Promise<unknown>
+
+describe('userPersistedStorage', () => {
+  const aliceClient = {
+    getStackClient: () => ({ uri: 'alice.mycozy.cloud' })
```

Comment:
this is not representative of a real life uri, you should have the protocol here
```suggestion
    getStackClient: () => ({ uri: 'https://alice.mycozy.cloud' })
```

Also you can add a test case with a port
```
    getStackClient: () => ({ uri: 'https://alice.cozy.tools:8080' })
```